diff -uNr dropbear-2020.81/default_options.h dropbear-2020.81-nx/default_options.h
--- dropbear-2020.81/default_options.h	2020-10-29 14:35:50.000000000 +0100
+++ dropbear-2020.81-nx/default_options.h	2022-02-27 10:05:39.770926156 +0100
@@ -19,10 +19,10 @@
 #define DROPBEAR_DEFADDRESS ""
 
 /* Default hostkey paths - these can be specified on the command line */
-#define DSS_PRIV_FILENAME "/etc/dropbear/dropbear_dss_host_key"
-#define RSA_PRIV_FILENAME "/etc/dropbear/dropbear_rsa_host_key"
-#define ECDSA_PRIV_FILENAME "/etc/dropbear/dropbear_ecdsa_host_key"
-#define ED25519_PRIV_FILENAME "/etc/dropbear/dropbear_ed25519_host_key"
+#define DSS_PRIV_FILENAME "/opt/ssh/dropbear_dss_host_key"
+#define RSA_PRIV_FILENAME "/opt/ssh/dropbear_rsa_host_key"
+#define ECDSA_PRIV_FILENAME "/opt/ssh/dropbear_ecdsa_host_key"
+#define ED25519_PRIV_FILENAME "/opt/ssh/dropbear_ed25519_host_key"
 
 /* Set NON_INETD_MODE if you require daemon functionality (ie Dropbear listens
  * on chosen ports and keeps accepting connections. This is the default.
@@ -269,11 +269,11 @@
  * OpenSSH), set the path below and set DROPBEAR_SFTPSERVER. 
  * The sftp-server program is not provided by Dropbear itself */
 #define DROPBEAR_SFTPSERVER 1
-#define SFTPSERVER_PATH "/usr/libexec/sftp-server"
+#define SFTPSERVER_PATH "/opt/ssh/sftp-server"
 
 /* This is used by the scp binary when used as a client binary. If you're
  * not using the Dropbear client, you'll need to change it */
-#define DROPBEAR_PATH_SSH_PROGRAM "/usr/bin/dbclient"
+#define DROPBEAR_PATH_SSH_PROGRAM "/opt/ssh/dbclient"
 
 /* Whether to log commands executed by a client. This only logs the 
  * (single) command sent to the server, not what a user did in a 
@@ -309,6 +309,6 @@
 #define DEFAULT_IDLE_TIMEOUT 0
 
 /* The default path. This will often get replaced by the shell */
-#define DEFAULT_PATH "/usr/bin:/bin"
+#define DEFAULT_PATH "/usr/bin:/bin:/opt/ssh:/opt/usr/nx-ks:/usr/sbin:/sbin"
 
 #endif /* DROPBEAR_DEFAULT_OPTIONS_H_ */
